modules/api/api-xsjs/src/main/java/com/sap/xsk/api/destination/CloudPlatform.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.api.destination;
+
+import com.google.gson.JsonObject;
+import com.sap.cloud.sdk.cloudplatform.ScpCfCloudPlatform;
+import com.sap.cloud.sdk.cloudplatform.exception.CloudPlatformException;
+import org.eclipse.dirigible.kyma.KymaModule;
+import org.eclipse.dirigible.commons.config.Configuration;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class CloudPlatform extends ScpCfCloudPlatform {
+
+  private static final String CLIENT_ID = "clientid";
+  private static final String CLIENT_SECRET = "clientsecret";
+  private static final String URL = "url";
+  private static final String URI = "uri";
+
+  @Nonnull
+  @Override
+  public JsonObject getServiceCredentials(@Nonnull String serviceName, @Nullable String servicePlan)
+      throws CloudPlatformException {
+    String destinationPrefix = Configuration.get(KymaModule.DIRIGIBLE_DESTINATION_PREFIX, "");
```

Comment:
these vars don't seem available in https://github.com/eclipse/dirigible/blob/master/releng/sap-kyma-base/src/main/java/org/eclipse/dirigible/kyma/KymaModule.java#L22 
Is there a branch or PR?